buildozer/def.bzl

```diff
@@ -1,4 +1,4 @@
-load("@bazel_skylib//:lib.bzl", "shell")
+load("@bazel_skylib//lib:shell.bzl", "shell")
```

Comment:
This and other similar changes solve the annoying warning that we get in each build.